README.md

```diff
@@ -1,2 +1,21 @@
+![Logo](Resources/logo-black-transparent.png)
 # Aquarius
-Contains the source code for Aquarius, a 2D game engine for the ECE 5010 junior software design project
+A 2D game engine developed by Colton Smith, Daniel Burke, & Liam Warren for the Junior Software Design Project (ECE 5010).
+
+## Additional Required Software
+- CMake Version 3.10 ([Download](https://cmake.org/download/))
+
+## Building Aquarius
+1. Unzip the ZIP File
+2. Navigate in command-line interface to the unzipped project folder 
+3. Make a build directory by executing the following command `mkdir build`
+4. Navigate to the build directory by executing the following command `cd build`
+5. Execute CMake to build the project by executing the following command `cmake ..`
+6. Generate an executable file by executing the following command `cmake --build . --config Release`
+
+## Running Aquarius
+1. Remain in the same command-line interface window as above, navigate to the Sandbox directory by executing 
+   the following command `cd Sandbox`    
```

Comment:
Navigate to build/Sandbox, this is a little confusing in case they close there console in between this section and the previous for some reason